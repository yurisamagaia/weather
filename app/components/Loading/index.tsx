import React, { FC } from 'react';
import { ActivityIndicator } from 'react-native';
import { colors } from '../../theme';

const Loading: FC = () => (
  <ActivityIndicator size="large" color={colors.white} />
);

export default Loading;