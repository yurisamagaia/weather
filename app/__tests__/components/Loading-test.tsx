import React from 'react';
import renderer from 'react-test-renderer';
import Loading from '../../components/Loading';

test('Should render correctly', () => {
  const tree = renderer.create(<Loading />).toJSON();
  expect(tree).toMatchSnapshot();
});

