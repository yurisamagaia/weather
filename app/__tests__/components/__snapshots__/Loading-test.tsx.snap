// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render correctly 1`] = `
<ActivityIndicator
  animating={true}
  color="#FFF"
  hidesWhenStopped={true}
  size="large"
/>
`;
